### PR TITLE
Reduced Hadoop version to ensure 2.8 compatibility

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -101,7 +101,7 @@
     <google.oauth-client.version>1.31.5</google.oauth-client.version>
     <google.protobuf.version>3.17.3</google.protobuf.version>
     <grpc.version>1.39.0</grpc.version>
-    <hadoop.two.version>2.10.1</hadoop.two.version>
+    <hadoop.two.version>2.8.0</hadoop.two.version>
```
Comment (medb) [inline pom.xml:104]: May we use 2.7.7 instead - it still widely used?

**Review by medb — CHANGES_REQUESTED** [review]

File: bigquery/pom.xml [hunk 2]
```diff
@@ -286,6 +292,7 @@
                   <include>io.grpc</include>
                   <include>io.opencensus</include>
                   <include>io.perfmark</include>
+                  <include>org.apache.avro</include>
```
Comment (medb) [inline bigquery/pom.xml:295]: Will this include avro in shaded jar? If so, we can not do this.

**Review by medb — CHANGES_REQUESTED** [review]

**Comment by mprashanthsagar** [comment]
@davidrabinowitz Is this PR still relevant ?

**Comment by davidrabinowitz** [comment]
Yes, but not urgent. It is to keep us supporting old Hadoop versions.

**Comment by medb** [comment]
Seems like it will be better to keep status quo than try to fix this, taking into account that this will be a maintenance branch anyway.

**Pull request closed without merging**